Sources/App/Commands/Analyze.swift

```diff
@@ -357,10 +357,11 @@ extension Analyze {
         guard let gitDirectory = Current.fileManager.cacheDirectoryPath(for: package.model) else {
             throw AppError.invalidPackageCachePath(package.model.id, package.model.url)
         }
-
+        
         repo.commitCount = (try? Current.git.commitCount(gitDirectory)) ?? 0
         repo.firstCommitDate = try? Current.git.firstCommitDate(gitDirectory)
         repo.lastCommitDate = try? Current.git.lastCommitDate(gitDirectory)
+        repo.authors = try PackageContributors.extract(gitCacheDirectoryPath: gitDirectory, packageID: package.model.id)
```

Comment:
@daveverwer: A number of the previous data extractions are using `try?` in order to ignore thrown errors. Are we happy that author extraction will completely throw the repo update if it fails?